app/konnect/analytics/services-and-routes.md

```diff
@@ -16,11 +15,11 @@ You can generate and export a CSV file for:
 
 * A service, including daily requests by status codes for all versions of the
```

Comment:
(Somewhat redundant with the first comment): we don't have charts for individual API product versions, just for an API product as a whole.  This chart is broken down by product version, not status code.  (The route graph is broken down by status code.)